src/views/token-swap/components/quote/quote.styles.js

```diff
@@ -72,22 +72,6 @@ const useQuoteStyles = createUseStyles((theme) => ({
     color: theme.palette.red.main,
     marginTop: theme.spacing(1),
   },
-  rewardCell: {
```

Comment:
this is from token swap, so it should stay. It's' a different kind of reward

Reply:
Thx! Reverted!